lib/src/default_index/revset_engine.rs

```diff
@@ -1047,6 +1047,22 @@ impl EvaluationContext<'_> {
                 let candidate_set = self.evaluate(candidates)?;
                 Ok(Box::new(self.take_latest_revset(&*candidate_set, *count)?))
             }
+            ResolvedExpression::Exactly { candidates, count } => {
+                let set = self.evaluate(candidates)?;
+                let positions: Vec<_> = set.positions().attach(index).try_collect()?;
```

Comment:
add a `.take(count + 1)` in there (but probably want `.saturating_add()`). otherwise `exactly(all(), 1)` at google might be slow